tests/test_live.py

```diff
@@ -222,9 +222,8 @@ def test_large_request(platform):
     )
 
 @pytest.mark.slow
-#@pytest.mark.parametrize("platform", ["staging-1-3", "staging-1-2", "production-1-2"])
-#def test_pickling(platform):
-def test_pickling():
+@pytest.mark.parametrize("platform", ["staging-1-3", "staging-1-2", "production-1-2"])
+def test_peculiar_request_causing_pickling_problem(platform):
```

Comment:
This test is not actually verifying anything? Is it supposed to be completed?

Reply:
This test is actually verifying very peculiar pickling problem!

Reply:
It is indeed completed.